Gemfile

```diff
@@ -6,12 +6,6 @@ ruby '3.1.3'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225283192-7426555b-8164-48f4-b6c6-18e3b264cf38.png)

**[Optional]**

- [ ] It would be nice if you make all your commit messages meaningful and capitalize to make your readme file descriptive and professional :+1: